modules/signing_scriptworker/templates/passwords-mobile.json.erb

```diff
@@ -10,5 +10,11 @@
     ],
     "project:mobile:fenix:releng:signing:cert:release-signing": [
         ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_fenix_prod_username"]) %>", "<%= scope.function_secret(["autograph_fenix_prod_password"]) %>", ["autograph_fenix"], "autograph"]
+    ],
+    "project:mobile:reference-browser:releng:signing:cert:release-signing": [
+        ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_reference_browser_prod_username"]) %>", "<%= scope.function_secret(["autograph_reference_browser_prod_password"]) %>", ["autograph_reference_browser"], "autograph"]
```

Comment:
This LGTM but we'll need to grab the credentials from SecOps and put them under `hiera` beforehand. @JohanLorenzo or myself can help with that.

Also, if we want the `reference-browser` APK be zipaligned, we might want to teach signingscript of its existence (e.g. https://github.com/mozilla-releng/signingscript/blob/master/signingscript/task.py#L31). If not, not need to touch `signingscript` at all as it'll automatically find it based on https://github.com/mozilla-releng/signingscript/blob/master/signingscript/sign.py#L83

Reply:
Mitch should now have the rights to put the creds into `hiera`. 

Good call on signingscript! We should eventually revisit the nomenclature to have a generic rule like `autograph_apk_*` to point to `sign_jar`. This way, we don't have to release a new version of signingscript every time we support a new browser.

Reply:
I've added:
* `autograph_reference_browser_prod_username`: `autograph_reference_browser_prod_username`
* `autograph_reference_browser_prod_password`: `encrypted(autograph_reference_browser_prod_password)`

to `secrets.eyaml` on the puppet master :)